Pod/Classes/BONTag.h

```diff
@@ -0,0 +1,69 @@
+//
+//  BONTag.h
+//  Pods
+//
+//  Created by Nora Trapp on 5/11/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "BONCompatibility.h"
+
+@protocol BONTextable;
+
+@interface BONTag : NSObject <NSCopying>
+
+@property (copy, nonatomic, readonly, BONNonnull) NSString *startTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *endTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *escapeString;
+@property (copy, nonatomic, readonly, BONNonnull) id<BONTextable> textable;
+
+- (BONNonnull instancetype)initWithStartTag:(BONNonnull NSString *)startTag endTag:(BONNonnull NSString *)endTag escapeString:(BONNonnull NSString *)escapeString textable:(BONNonnull id<BONTextable>)textable;
```

Comment:
I was thinking of `NS_SWIFT_NAME`, which lets you describe the function signature, but doesn't appear to let you control things like default parameters. Swift subspec would be nice, but more likely I'll just wait, and rewrite BonMot in Swift for 4.0 or 5.0, once including a Swift dependency in a project is no longer a hardship.